### ISSUE TITLE
Editor is blocked two or three second

### ISSUE THREAD

**Issue by pabloalmunia** [vsch/idea-multimarkdown#838]
- Webstorm: 2020.1
- Multimarkdown: 3.0.201.79
- OS: Windows 10 pro (64 bits)
- CPU: Intel(R) Core(TM) i7-6700HQ CPU @ 2.60GHz
- Mem: 16 GB
- Webstorm Map Head Size: 4029M

When I change between Markdown windows, the editor is blocked two or three seconds before I can write. Typically, my .md files has between 5Kb to 30Kb, no more.

**Comment by vsch** [comment]
@pabloalmunia, is it blocked or the text shows as plain text without syntx highlighting for a couple of seconds?

**Comment by pabloalmunia** [comment]
It's complety blocked, I cannot write or move the cursor or click over other position into the text.

**Comment by pabloalmunia** [comment]
![multimarkdown](https://user-images.githubusercontent.com/1365303/79047135-1e232a80-7c15-11ea-98fb-680d4f9aab54.gif)

This is a simple example. After the page change, I try to click on the title, but it doesn't respond until several seconds later.

**Comment by vsch** [comment]
@pabloalmunia, I am not able to reproduce this on my test system with Windows 10, WebStorm 2020.1 and Markdown Navigator 3.0.201.79. 

**Q: Do you have preview enabled?
Q: Do you update your IDE through JetBrains Toolbox or manually through the IDE?**

**Comment by pabloalmunia** [comment]
Q: Do you have preview enabled?
No
Q: Do you update your IDE through JetBrains Toolbox or manually through the IDE?
from the IDE

**Comment by vsch** [comment]
@pabloalmunia, in my experience updating the IDE manually had a tendency to make it unstable after a few updates, especially EAP.

I cannot claim that this is the problem but I am not able to replicate this issue and suspect that there is something out of whack.

It is also possible that it is an interaction between plugins. The two somehow cause a UI lock up for a couple of seconds.

Are you getting any errors in the logs? The IDE now generates errors in the log if the UI freezes, which sounds like what you are getting.

**Comment by pabloalmunia** [comment]
Well, I have advanced work and I had already completely uninstalled WebStorm and deleted absolutely all the configuration folders.

I have installed Webstorm and have only installed and activated the minimal set of plugins.

As a result I have a more stable behavior. I have a small deleay of few seconds the first time when I open an .md file, but after this first open the response is immediate.

**Comment by vsch** [comment]
@pabloalmunia, that is good news.

I used to update my IDEs manually, but noticed that after a few updates the IDE started developing all kinds of issues which would go away if I uninstalled it, backed up the settings and then deleted all its support directories, then re-installed it fresh.

Since I adopted using JetBrains Toolbox which updates the IDEs, I never have experienced these stability issues. It also gives you one place to upgrade/install/uninstall all your JetBrains IDEs, including managing multiple versions.

I would recommend installing JetBrains toolbox to manage the IDEs.
